.github/workflows/ci.yml

```
@@ -0,0 +1,275 @@
+name: CI Pipeline
+
+on:
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)